flathub.json

```diff
@@ -0,0 +1,6 @@
+{
+  "only-arches": [
+    "x86_64"
```

Comment:
this is disappointing.

Reply:
I couldn't find a way to emulate an arm64 linux environment to build for flutter arm64 on my machine as of yet. If I find a way to do that in the future, I will add arm64 builds.

Reply:
the permanent disappointement that flutter is.